src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -253,10 +253,10 @@ internal enum WellKnownType
         System_ValueTuple_T3,
         System_ValueTuple_T4,
         System_ValueTuple_T5,
-        System_ValueTuple_T6,
 
         ExtSentinel, // Not a real type, just a marker for types above 255 and strictly below 512
 
+        System_ValueTuple_T6,
```

Comment:
Why did this change need to be made? Did we just get the marker wrong?

Reply:
> Why did this change need to be made? Did we just get the marker wrong?

A new type was added in front of the marker, special types precede all well-known types. The marker, however, should maintain its value (255)

---
In reply to: [250377270](https://github.com/dotnet/roslyn/pull/32689#discussion_r250377270) [](ancestors = 250377270)